test/test-execute/exec-bind-paths.service

```diff
@@ -0,0 +1,14 @@
+[Service]
+Type=oneshot
+# Create a file in /tmp/test-exec_bind_paths
+ExecStart=/bin/touch /tmp/test-exec_bind_paths/thisisasimpletest
+# Then, the file can be access through /tmp
+ExecStart=/bin/test -f /tmp/thisisasimpletest
```

Comment:
I have a suspicion that there is no `test` in `/bin`, so `/bin/test` should probably be replaced by `/bin/sh -c 'test ...'`.

Reply:
Thanks.